Dockerfile

```diff
@@ -1,34 +1,48 @@
-FROM node@sha256:dbb1e74774617c46a34f67d25decbdc5841af2a2a08a125a37743c734799b909 as builder
-# Docker Tag: node:8.11.1-stretch
+FROM parity/parity:v1.9.7
 
-ADD ./vendor /tmp/vendor
+ADD ./key.json /tmp/keystore/
+ADD ./setup_8.x /tmp/
+
+RUN bash /tmp/setup_8.x
 
 WORKDIR /build
 
-# install vendor'd tools and dependencies
-RUN dpkg -i /tmp/vendor/*.deb
```

Comment:
Curious why the switch from vendored to apt-get?  Simpler for MVP?  Long term, we should figure out a solution that lets us have deterministic builds.  I'm OK with this for MVP and/or while iterating though as I recognize that vendoring everything is a bit of a PITA early on.

Reply:
Only switched this while developing. Once we get a sense for packages we want, i'll re-vendor them